com.reprezen.swagedit.tests/src/com/reprezen/swagedit/editor/outline/OutlineStyledLabelProviderTest.xtend

```diff
@@ -44,18 +42,62 @@ class OutlineStyledLabelProviderTest {
 			  - https
 		'''
 
-		val els = OutlineElement.create(yaml.compose(new StringReader(text)))
+		val els = Model.parseYaml(text).root
 
 		assertEquals("schemes", els.get(0).text)
-		assertEquals(els.get(0).text, provider.getSyledString(els.get(0)).toString)
+		assertEquals("schemes schemesList", provider.getSyledString(els.get(0)).toString)
 		
-		val http = els.get(0).children.get(0)
+		val http = els.get(0).elements.get(0)
 		assertEquals("http", http.text)
-		assertEquals(http.text, provider.getSyledString(http).toString)
+		assertEquals("http", provider.getSyledString(http).toString)
 		
-		val https = els.get(0).children.get(1)
+		val https = els.get(0).elements.get(1)
 		assertEquals("https", https.text)
-		assertEquals(https.text, provider.getSyledString(https).toString)
+		assertEquals("https", provider.getSyledString(https).toString)
+	}
+
+	@Test
+	def void should_Display_Name_And_Type_For_Single_Valued_Objects() {
+		val text = '''
+			object:
+			  name: hello
+			  value: world
+		'''
+
+		val els = Model.parseYaml(text).root
+		assertEquals("object", els.get(0).text)
+
+		// TODO
+	}
+
+	@Test
+	def void should_Display_Type_For_Arrays_Of_Objects() {
+		val text = '''
+			objects:
+			  - name: hello
+			    value: world
+			  - name: hello
+			    value: world
+		'''
+		
+		val els = Model.parseYaml(text).root
+		assertEquals("objects", els.get(0).text)
+		
+		// TODO
```

Comment:
Let's create an issue for the TODO items, otherwise we can easily forget about them